README.md

```diff
@@ -65,6 +65,21 @@ Tests for this contract are implemented in Haskell using the [cleveland framewor
 For more detail about the test suite and how to execute it, check out its
 [README](./haskell/test/README.md).
 
+## Contract Metadata
+
+BaseDAO metadata should be compliant with [TZIP-16](https://gitlab.com/tzip/tzip/-/blob/master/proposals/tzip-16/tzip-16.md).
+
+It can be generated via:
+
+```bash
+cd haskell
+stack exec -- baseDAO-ligo-meta print-metadata > metadata.json
+```
+
+This metadata should be stored via:
+- As a separate contract: [The `tezos-storage` URI Scheme](https://gitlab.com/tzip/tzip/-/blob/master/proposals/tzip-16/tzip-16.md#the-tezos-storage-uri-scheme)
+- On IPFS file storage: [Get started with IPFS](https://ipfs.io/)
```

Comment:
Given that this is the main point of this issue/PR, I would put this as the first option.

Also `FS` in `IPFS` stand for `file storage`, so I think this can be rephrased to avoid this repetition.